src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -420,11 +424,15 @@ class Project: ADOSVTBase
                 $this.PAMembers = @($this.PAMembers | Select-Object displayName,mailAddress)
                 if($TotalPAMembers -lt $this.ControlSettings.Project.MinPAMembersPermissible){
                     $controlResult.AddMessage([VerificationResult]::Failed,"Number of administrators configured are less than the minimum required administrators count: $($this.ControlSettings.Project.MinPAMembersPermissible)");
-                    $controlResult.AdditionalInfoInCSV += "TotalPA: $($TotalPAMembers); ";
-                    $controlResult.AdditionalInfoInCSV += "MinPA: $($this.ControlSettings.project.MinPAMembersPermissible);";
+                    $controlResult.AdditionalInfoInCSV += "NumPAs: $($TotalPAMembers); ";
+                    $controlResult.AdditionalInfoInCSV += "MaxPAAllowed: $($this.ControlSettings.project.MaxPAMembersPermissible); ";
+                    $controlResult.AdditionalInfoInCSV += "MinPAReqd: $($this.ControlSettings.project.MinPAMembersPermissible);";
                 }
                 else{
                     $controlResult.AddMessage([VerificationResult]::Passed,"Number of administrators configured meet the minimum required administrators count: $($this.ControlSettings.Project.MinPAMembersPermissible)");
+                    $controlResult.AdditionalInfoInCSV += "NumPAs: $($TotalPAMembers); ";
```

Comment:
As same lines are getting repeated for both pass and fail scenario, can this be added just once in the end?

Also, line 448 has older additional info

Reply:
i am removing from all other places and just adding it at the end while returning controlResults.